x-pack/plugins/ml/common/util/es_utils.test.ts

```diff
@@ -52,7 +52,26 @@ describe('Util: isValidIndexName()', () => {
     expect(isValidIndexName('a'.repeat(255))).toBe(true);
```

Comment:
@elastic/ml-ui I broke this test as I have now added TextEncoder to our jest config, I've now added a test case for when TextEncoder is present and missing 👍